modules/builds-disabling-build-strategy-globally.adoc

```diff
@@ -49,45 +25,14 @@ subjects:
 $ oc adm policy remove-cluster-role-from-group system:build-strategy-docker system:authenticated
 ----
 
-. Ensure the build strategy subresources are also removed from these roles:
+. Ensure the build strategy subresources are also removed from the `admin` and `edit` user roles:
 +
 [source,terminal]
 ----
-$ oc edit clusterrole admin
+$ oc get clusterrole admin -o yaml | grep "builds/docker"
```

Comment:
Usually we'd want to only see one command block per step (i.e., break this into descriptive substeps) and ideally have some example output, but I think this is fine for now considering urgency.